localtileserver/client.py

```diff
@@ -89,11 +89,17 @@ def band_names(self):
 
     @property
     def min_zoom(self):
-        return self.info.minzoom
+        if hasattr(self, "info") and hasattr(self.info, "minzoom"):
```

Comment:
the first hasattr check here isn't necessary as the client has info as a defined property. Just the secod check is all that is needed

Reply:
It has been fixed. I wasn't sure if rio-tiler has removed the info attribute entirely

Reply:
Oh, I see what you mean. I’ll take a look at this

Reply:
The Info class is still present, just min/max zoom attributes were removed.